### DISCUSSION TITLE
Play rate (speed) & pitch change

### DISCUSSION THREAD

**Discussion by plebcrabslayer** [SubtitleEdit/subtitleedit#5333] [Q&A] [answered]
Hello. I hope that this is the right, or an acceptable, place to post this problem/question.
My "home" version (on PC, 3.6.2) of Subtitle Edit maintains the pitch of the audio when I slow it down or speed it up using the "Play rate (speed)" drop-down menu beneath the waveform (and next to the zoom drop-down menu).
Now that I am away from home, I had to download Subtitle Edit onto my laptop (also 3.6.2). When I select a non-100% play rate, the pitch changes accordingly. This is useless and completely defeats the purpose.
I cannot find a single option to change this. Is there one that I am simply not seeing?
Cheers!

**Comment by niksedk** [comment, marked as answer]
I think it depends on the video player... could you try mpv?

**Reply by plebcrabslayer** [reply]
I don't use video files. I time the subtitles directly from an mp3, so I use only the waveform. Could it be codec-related and I'm simply missing one on my laptop that would allow "pitch-uniformity"?

**Reply by niksedk** [reply]
You still need a "media player"... e.g. mpv - try Options - Settings - Video player - download/choose mpv

(I guess you're using mpv or vlc on your home computer, and direct show on your laptop. So yes, it's codec/player related)

**Reply by plebcrabslayer** [reply]
Brilliant! That's exactly what it was, and now slo-mo audio maintains the original pitch! Thank you very much. :)
